src/Symfony/Component/Notifier/Event/SentMessageEvent.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Event;
+
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * @author Smaïne Milianni <smaine.milianni@gmail.com>
+ */
+final class SentMessageEvent extends Event
```

Comment:
this class extends `Event` but the Failure Event does not.

Reply:
nice catch :eye: 
fixed in https://github.com/symfony/symfony/pull/39601/commits/5970563938dc2a1eba6a15a43289935d9fc9f17e